webapp/view/ListReport.view.xml

```diff
@@ -0,0 +1,187 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:f="sap.f"
+	xmlns:core="sap.ui.core"
+	xmlns:fb="sap.ui.comp.filterbar"
+	controllerName="products.app.controller.ListReport">
+
+		<f:DynamicPage 
+			id="dynamicPageId" 
+			preserveHeaderStateOnScroll="true" 
+			showFooter="true" 
+			fitContent="true">
+			<!-- DynamicPage Title -->
+			<f:title>
+				<f:DynamicPageTitle>
+					<f:heading>
+						<Title text="{i18n>ListReportTitle}"/>
+					</f:heading>
+					
+				</f:DynamicPageTitle>
+			</f:title>
+
+			<!-- DynamicPage Header -->
+			<f:header>
+				<f:DynamicPageHeader>
+					
+					<f:content>
+						
+						<fb:FilterBar id="filterbar" persistencyKey="myPersKey" useToolbar="false" >
+							<fb:filterGroupItems >
+
+								<fb:FilterGroupItem 
+									name="search" 
+									label="{i18n>NameText} {i18n>AndText} {i18n>DescriptionText}" 
+									groupTitle="false" 
+									groupName="HeaderFiltersGroup" 
```

Comment:
Always think about how people could understand your code. Header Filters Group is good, it shows that a field belongs to the filtering group and it can change how data appears. But you also could provide more details, by renaming it to ProductFilterGroup. So it would be easier to understand its domain, I think